src/main/scala/tech/cryptonomic/conseil/util/JsonUtil.scala

```diff
@@ -56,7 +56,7 @@ object JsonUtil {
     lazy val emptyObject = JsonString("{}")
 
     /** add standard cleaning for input json */
-    def sanitize(s: String): String = s.filterNot(_.isControl).replaceAll("\\\\", "backslash")
+    def sanitize(s: String): String = s.filterNot(_.isControl).replaceAll("""\\u""", "backslashu")
```

Comment:
We might actually want to use a regex here to capture only unicode codepoints
Something like
`replaceAll("""\\u(0-9a-fA-F){1,4}""", "backslashu")`

We're loosing the actual unicode value but it might be a good start?

Reply:
backslashu?